### PR TITLE
Adding Peaks.js showing dummy data in the waveform

### PR DESCRIPTION
**Pull request by Dananji**

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -72,3 +65,54 @@ section {
...
+  width: 100%;
+}
+
+[id*="waveform-visualiser"] [class*="-container"] {
+  box-shadow: 3px 3px 20px #919191;
+  margin: 0 0 24px 0;
+  -moz-box-shadow: 3px 3px 20px #919191;
+  -webkit-box-shadow: 3px 3px 20px #919191;
+  line-height: 0;
+}
```
Comment (adamjarling) [inline src/App.css]: In general, is it possible to use direct class names in CSS references?   Once you get into Hyrax code, the Hound code quality tool will complain about CSS declarations like this...  just a heads up.

File: src/components/Waveform.js [hunk 2]
```diff
@@ -1,9 +1,89 @@
...
+          },
+		  keyboard: true,
+		  pointMarkerColor: '#006eb0',
+		  showPlayheadTime: true
+		};
+
+		this.setState({
+			peaksInstance: Peaks.init(options)
+		});
+	}
```
Comment (adamjarling) [inline src/components/Waveform.js]: Generally, in React using `document.getElementById()` and other vanilla JS targeting is an anti-pattern.  To get this through, and if it works that's fine, however check out React's `ref`.  It's the "React way" of targeting an element when you need a direct reference to a DOM element.

File: src/App.css (hunk 1 above)
Comment (Dananji) [reply src/App.css]: It is possible using [attribute*=value] selector in CSS. I guess it is not a good way of doing this. I will change this to a more generic approach using something like an HTML class.

Comment (adamjarling) [reply src/App.css]: Hi @Dananji .So best practice for CSS, and CSS linters (which will complain about certain things), is to use CSS class selectors in almost all cases.  The preferred format is something like `something-wrapper`, or `something-button-group`, `something-button`....to that effect.

**Pull request closed without merging**